bluzelled/src/main.cpp

```diff
@@ -1,54 +1,39 @@
 #include "Node.h"
 #include "NodeUtilities.h"
 #include "web_sockets/WebSocketServer.h"
-#include "web_sockets/WebSocket.h"
-#include "web_sockets/Listener.h"
-#include "web_sockets/Session.h"
-#include "server/server.hpp"
+#include "http/server/ServiceManager.h"
 #include "NodeManager.h"
 
 #include <boost/exception/all.hpp>
-#include <boost/beast/core.hpp>
 #include <boost/beast/http.hpp>
-#include <boost/beast/version.hpp>
-#include <boost/asio/ip/tcp.hpp>
-#include <boost/asio/strand.hpp>
-#include <boost/config.hpp>
-#include <iostream>
-#include <string>
 
 static unsigned long s_max_nodes = 25;
 static unsigned long s_min_nodes = 5;
 static Nodes s_nodes;
```

Comment:
I a not big fan of defines for types. It's not clear what Nodes is. It looks like a container but is it vector, map, always have to go to definition to figure out.